package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "the-travel-shelf",
+  "version": "1.0.0",
+  "description": "A magazine and index of beautiful global independent bookstores",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/RichardLitt/the-travel-shelf.git"
+  },
+  "keywords": ["independent", "bookstores", "index"],
+  "author": "Richard Littauer",
+  "license": "MIT",
+  "contributors": ["Joanna Katsifaraki", "Jose Esplana", "Hanie Jafari, Vernon Lillies" ],
```

Comment:
I believe these need to include more information. Take a look at some other package.json's that have it. You can find them through GitHub search. Same for the `author` field.